common/djangoapps/external_auth/views.py

```diff
@@ -175,7 +175,7 @@ def _external_login_or_signup(request,
                     failure_msg = _(dedent("""
                         You have already created an account using an external login like WebAuth or Shibboleth.
                         Please contact %s for support """
-                                           % getattr(settings, 'TECH_SUPPORT_EMAIL', 'techsupport@class.stanford.edu')))
+                                           % getattr(settings, 'TECH_SUPPORT_EMAIL', 'techsupport@stanfordonline.zendesk.com')))
```

Comment:
Is `TECH_SUPPORT_EMAIL` ever undefined?

I know the edx code base is littered with calls like `getattr(settings, 'KEY', 'default')`, but those settings are _always_ defined and _never_ need the default. If we know there's a value stored in either `common.py` or `aws.py` or `envs.json`, we can just write `settings.TECH_SUPPORT_EMAIL`.

Also, that means we'll never have to change this line in future :)